File: app/src/main/java/com/mgaetan89/showsrage/fragment/SettingsAboutShowsRageFragment.java
package com.mgaetan89.showsrage.fragment;

import com.mgaetan89.showsrage.R;

public class SettingsAboutShowsRageFragment extends SettingsAboutFragment {
	@Override
	protected int getTitleResourceId() {
		return R.string.app_name;
	}

	@Override
	protected int getXmlResourceFile() {
		return R.xml.settings_about_showsrage;
	}
}

File: app/src/main/kotlin/com/mgaetan89/showsrage/fragment/SettingsAboutShowsRageFragment.kt
package com.mgaetan89.showsrage.fragment

import com.mgaetan89.showsrage.R

class SettingsAboutShowsRageFragment : SettingsAboutFragment() {
	override fun getTitleResourceId() = R.string.app_name

	override fun getXmlResourceFile() = R.xml.settings_about_showsrage

	companion object {
		fun newInstance() = SettingsAboutShowsRageFragment()
	}
}
